app/templates/common/_nav.html

```diff
@@ -8,7 +8,7 @@
             <div class="collapse navbar-collapse" id="navbarSupportedContent">
                 <ul class="navbar-nav ms-auto">
                     <li class="nav-item display-lg display-md">
-                        {% if url_for(request.endpoint) == '/' %}
```

Comment:
This was blocking the profile page from rendering as it cant fetch routes that require route params this way. Instead just check for the function it calls